src/aof.c

```diff
@@ -1827,13 +1815,13 @@ int loadAppendOnlyFiles(aofManifest *am) {
         last_file = ++aof_num == total_num;
         start = ustime();
         ret = loadSingleAppendOnlyFile(aof_name);
-        if (ret == AOF_OK || ((ret == AOF_TRUNCATED || ret == AOF_BROKEN_RECOVERED) && last_file)) {
+        if (ret == AOF_OK || (ret == AOF_TRUNCATED && last_file)) {
```

Comment:
please mention this change in the top comment. 
Why is it not needed now? What changes would there be without it?